src/python/pants/backend/python/lint/pylint/subsystem.py

```diff
@@ -77,7 +77,7 @@ class Pylint(PythonToolBase):
     name = "Pylint"
     help = "The Pylint linter for Python code (https://www.pylint.org/)."
 
-    default_version = "pylint>=2.11.0,<2.12"
```

Comment:
Let's keep this range narrower, so that it captures the version that is actually being used. Also, we should update the title of the PR with the version which is actually pinned in the lockfile... looks like `2.13.9`.